packages/react/src/AnchorNav/AnchorNav.module.css

```diff
@@ -0,0 +1,339 @@
+/*
+ * AnchorNav
+ * 
+ * 1. Globals
+ * 2. AnchorNav root and inner bounding containers
+ * 3. AnchorNav links
+ * 4. AnchorNav link labels
+ * 5. AnchorNav menu toggle button (narrow)
+ * 6. AnchorNav action
+ * 7. Custom animations
+ */
+
+/*
+ * 1. Globals
+ * TODO: Move into global reset
+ */
+:global(html) {
+  scroll-behavior: smooth;
+}
+
+@media screen and (prefers-reduced-motion: reduce) {
+  :global(html) {
+    scroll-behavior: auto;
+  }
+}
+
```

Comment:
I added this for progressive enhancement reasons, just in case someone wanted to use this stylesheet outside of a React component, it still behaves the same. 

That said, I'm happy to remove it too. I think I added a TODO and asked if we should move this into global reset.css instead. What do you think @danielguillan?